src/style.css

```
@@ -0,0 +1,26 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+/* #region stars__star */
```

make sure to remove all the unnecessary comments before commiting

---

> 

```suggestion
/* #region stars__star */
```
So shouldn't I use this to wrap the CSS selector ever?